tests/snapshots/copy.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
It's not really clear what these shell scripts are being added for.  Are these things people would be expected to use?  Can they include some comments at least providing some information on what they are for?

Reply:
Sure, I'll had comments.